lib/gas_adjuster/src/tests.rs

```diff
@@ -0,0 +1,23 @@
+use super::GasStatisticsInner;
```

Comment:
If you will move gas statistics to a separate module, it makes sense to move the tests to the inline module within this file.